### PR TITLE
Show the link to the All Hands product roadmap

### PR DESCRIPTION
**Pull request by ryx2**
**End-user friendly description of the problem this fixes or functionality that this introduces**

- [ ] Include this change in the Release Notes. If checked, you must provide an **end-user friendly** description for your change below

---
**Give a summary of what the PR does, explaining any non-trivial design decisions**

Shows people the all hands public roadmap, so open source contributors can hopefully tailor their changes to align with the openhands maintainers. 

---
**Link of any specific issues this addresses**

### CODE REVIEW

**Comment by ryx2** [comment]
https://github.com/All-Hands-AI/OpenHands/issues/5191

**Comment by ryx2** [comment]
https://github.com/All-Hands-AI/OpenHands/issues/5191

**Review by neubig — CHANGES_REQUESTED** [review]
Let's change "All Hands product roadmap"  to "Openhands roadmap", and move it to the top of the "progress" section.

**Review by rbren — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -88,6 +88,8 @@ through Slack, so this is the best place to start, but we also are happy to have
 
 See more about the community in [COMMUNITY.md](./COMMUNITY.md) or find details on contributing in [CONTRIBUTING.md](./CONTRIBUTING.md).
 
+See the monthly All Hands product roadmap [here](https://github.com/orgs/All-Hands-AI/projects/1) (updated at the maintainer's meeting at the end of each month).
+
 ## 📈 Progress
```
Comment (neubig) [inline README.md:93]:
```suggestion
## 📈 Progress

See the monthly OpenHands roadmap [here](https://github.com/orgs/All-Hands-AI/projects/1) (updated at the maintainer's meeting at the end of each month).
```

**Review by neubig — APPROVED** [review]
Fixed so this should be ready to merge!

**Comment by ryx2** [comment]
Any thoughts on also putting this in the "contributing" md?

**Pull request merged**